src/app/portlets/content-types/create/content-types-create.component.html

```diff
@@ -3,6 +3,7 @@
         [type]="contentTypeType"
         [name]="contentTypeName | async"
         [icon]="contentTypeIcon"
+        [onCreate]="true"
```

Comment:
The content-types-form has a data property, this property is null for create mode so I think that we dont need a new property.